contracts/cosmwasm-vm/cw-ibc-core/src/ics04_channel/packet/timeout.rs

```diff
@@ -351,11 +351,10 @@ impl<'a> CwIbcCoreContext<'a> {
 
                     let event = Event::new(IbcEventType::Timeout.as_str())
                         .add_attribute("channel_order", chan_end_on_a.ordering().as_str());
-                    let mut events = vec![event];
+                    let events = vec![event];
                     if let Order::Ordered = chan_end_on_a.ordering {
-                        let close_init =
-                            create_close_init_channel_event(port_id.as_str(), channel_id.as_str());
-                        events.push(close_init);
+                        // let close_init = create_packet_timeout_event(,chann);
+                        // events.push(close_init);
```

Comment:
Why is this commented ?

Reply:
the logic seems wrong because the packet sequence is saved only after light client validation but its trying to fetch it right after validation which causes error.